test/controllers/fsapi_controller_test.rb

```diff
@@ -206,7 +206,9 @@ def test_caller_id_number_hidden
 
   def test_pin_prompt_allotted_timeout_not_set
     Rails.configuration.x.stub(:fsapi_max_duration, 0) do
-      post(fsapi_url, params: { section: 'dialplan', 'Caller-Destination-Number': '5551234' })
+      FsapiController.stub_any_instance(:authenticate, true) do
+        post(fsapi_url, params: { section: 'dialplan', 'Caller-Destination-Number': '5551234' })
+      end
```

Comment:
For those two tests, the requests are not authenticated, which causes an (unauthorized) error.

Not sure if this is wanted or not, as all other test have authenticated requests.